test/tests.js

```diff
@@ -1 +1,5 @@
 import './functions.test.js';
+import './functions.js';
```

Comment:
I see what you're doing here, but I'd say nuke this to keep this file clean. The function works, write your test in functions.test.js!